recipes/gn/meta.yaml

```diff
@@ -0,0 +1,45 @@
+# As there are no releases in GN, we build the latest tag from time to time.
+# The version is the time of the latest master commit in YYYYMMDDHHMMSS
+# notation.
+
+{% set name = "gn" %}
+{% set version = "0.0.20190712172645" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # We have to use git as the source as the provided tarballs are not stable.
+  git_url: https://gn.googlesource.com/gn
```

Comment:
Not ideal but hey, it is Google, we cannot really expect them to make OSS life easy right ;-p
(Sorry for the rant, just package jax/jaxlib from Google and it was a terrible experience.)